components/slider/index.jsx

```diff
@@ -140,13 +139,13 @@ class Slider extends React.Component {
 
 	handleChange = (event) => {
 		if (isFunction(this.props.onChange)) {
-			this.props.onChange(event, { value: event.target.value });
+			this.props.onChange(event, { value: parseInt(event.target.value, 10) });
```

Comment:
This could pose a problem should a user want to make a slider for floats (a slider between 0 and 1 for example).  I'm not sure we want to have `parseInt` here because we can't assume.

Reply:
value return text, but we expect number be passed in. I'll look into using `Number()`.